src/modules/45-dashboards/components/DashboardCard/DashboardCard.tsx

```diff
@@ -42,6 +40,15 @@ const DashboardCard: React.FC<DashboardCardProps> = ({
   const { getString } = useStrings()
   const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
   const [menuOpen, setMenuOpen] = useState(false)
+  const { editableFolders } = useDashboardsContext()
+
+  const isCloneable: boolean = React.useMemo(() => {
+    return editableFolders ? editableFolders?.length > 0 : false
+  }, [editableFolders])
+
+  const isEditable: boolean = React.useMemo(() => {
+    return editableFolders?.some(folder => folder.id === dashboard.folder.id) ?? false
```

Comment:
You can simplify here as well by just casting to a boolean
```suggestion
    return !!editableFolders?.some(folder => folder.id === dashboard.folder.id)
```